app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CustomersController < ApplicationController
+  def show
+    @customer = Customer.find(params[:id])
+  end
+
+  def update 
```

Comment:
A more RESTful way to build this controller action would be to consider a 3rd controller, the CustomerItemsController and use a create action there to indicate that you are creating a relationship between a customer and an item.

For an application of this size, this is not a big deal, but imagine in a larger application, you might want to reserve the customers#update to an action that just updates the attributes of a customer (i.e. name, etc).